### PR TITLE
You've just reached 10,000 dependent modules.

### PR DESCRIPTION
**Pull request by popey456963**
:confetti_ball: :tada:  Congratulations! :tada: :confetti_ball:

### CODE REVIEW

**Comment by Qix-** [comment]
Sure enough! 💃 🎉 🎂 🎈

File: readme.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
 - Clean and focused
 - Auto-detects color support
 - Actively maintained
-- [Used by ~7700 modules](https://www.npmjs.com/browse/depended/chalk) as of March 15, 2016
+- [Used by ~10000 modules](https://www.npmjs.com/browse/depended/chalk) as of Augest 2nd, 2016
```
Comment (Qix-) [inline readme.md]: Mind adding a comma? `10,000`

**Comment by popey456963** [comment]
Altered to include a beautiful comma :smile:

**Comment by Qix-** [comment]
Thank you ^_^

**Comment by sindresorhus** [comment]
Wee 😁

**Pull request merged**